### PR TITLE
Scrollbar buttons are removed

### PR DESCRIPTION
**Pull request by Sabbirfeni**
## Fixes Issue
Closes #565

## Changes proposed
App.css file has been updated. New code added in order to hide the unexpected scrollbar buttons.

## Screenshots
scrollbar

## Note to reviewers
Please check out the changes.

## Code of Conduct

- [✔] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/thecyberworld/Support/blob/main/CODE_OF_CONDUCT.md) 🖖

## Check List (Check all the applicable boxes) 

- [✔] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [✔] All new and existing tests passed.
- [✔] This PR does not contain plagiarized content.
- [✔] The title of my pull request is a short description of the requested changes.

---

You can also join our [Discord](https://discord.gg/QHBPq6xP5p) community.
Feel free to check out other cool repositories of the [Thecyberworld](https://github.com/thecyberworld).
Join the Thecyberworld GitHub Organisation by raising an [issue](https://github.com/thecyberworld/Support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) (you will be sent an invitation).

### CODE REVIEW

**Comment by Sabbirfeni** [comment]
@ArkadiK94 

Please check out this PR.

**Comment by kabir0x23** [comment]
hi, can u add a screenshot?

File: src/App.css [hunk 1]
```diff
@@ -31,6 +31,11 @@ html {
     width: 8px;
     height: 3px;
 }
+
+::-webkit-scrollbar-button {
+    display: none; /* This will hide the scrollbar buttons */
+}
+
 ::-webkit-scrollbar-button {
```
Comment (ArkadiK94) [inline src/App.css:39]: you have two selectors that are the same, so if you display:none; you can remove the next selector which indicates the background-color.

**Review by ArkadiK94 — CHANGES_REQUESTED** [review]
Hi, thanks for contributing, please, change the code. Also, as @kabir0x23 said add a screen shot or short video. I am using the "snipping tool" in windows.

File: src/App.css (hunk 1 above)
Comment (Sabbirfeni) [reply src/App.css:39]: Okay.

**Comment by Sabbirfeni** [comment]
@ArkadiK9 I have updated the and opened new pull request. And here is the screenshot.
scrollbar

**Review by ArkadiK94 — APPROVED** [review]
Great, thanks for your contribution 👍

**Comment by ArkadiK94** [comment]
@kabir0x23 Could you merge this?

**Pull request merged**